app/User.php

```diff
@@ -154,7 +154,7 @@ public function threads()
      */
     public function latestThreads(int $amount = 3)
     {
-        return $this->threadsRelation->take($amount);
+        return $this->threadsRelation()->limit($amount)->get();
```

Comment:
I did a mistake. But still we have to give like this

`return $this->threadsRelation()->latest()->limit($amount)->get();`